content/en/docs/concepts/cluster-administration/node-shutdown.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Node Shutdowns
+api_metadata:
+- apiVersion: "v1"
+  kind: "Node"
```

Comment:
I would not have this here; instead make “node” in the introduction be a glossary tooltip.

Here's why: we don't have an API reference for _node shutdowns_. We actually might one day though, see https://github.com/kubernetes/enhancements/issues/4212

People can click on the glossary tooltip and then with one more click see the API reference, obviously connected to the topic (nodes) that it's the API reference for.

Reply:
Done. Thank you for the detailed explanation